README.md

```diff
@@ -28,17 +28,13 @@ If this command fails, check out [Troubleshooting](#troubleshooting) for suggest
 - Local_Video_YourName
 - Local_High_Priority_YourName
 
-4. Export AWS configuration variables.
+4. Define environment variables in `.env` file.
```

Comment:
This is great, and actually what I'm already doing!

Unfortunately, that means that checking out this branch conflicts with my local file, and will conflict with any local changes in the future, which these instructions are encouraging.

Instead, the pattern I've successfully used before is to check in a `.env.template` file with the relevant variables filled with example data, and have the instructions include a step like `cp .env.template .env`. You already made the template, and you've already added `.env` to the `.gitignore`, so all that needs to change is the template name and the instructions.

Here's a draft of the new instructions, in the hopes that it is helpful:

```suggestion
4. Define environment variables in `.env` file. Copy the template file (`cp .env.template .env`), and open the copy in your preferred editor.
```